frontend/src/App.tsx

```diff
@@ -37,6 +42,13 @@ type IAllAppProps =
   IAppProps &
   DispatchProp;
 
+interface IBackendUser {
+  success: boolean;
+  message: string;
+  user: IUser;
+  cookies: any;
+}
+
```

Comment:
Ideally, these would be defined/shared between the frontend/backend.  As they really need to stay in sync. Right?